config/initializers/devise.rb

```diff
@@ -6,7 +6,7 @@
   # confirmation, reset password and unlock tokens in the database.
   # Devise will use the `secret_key_base` on Rails 4+ applications as its `secret_key`
   # by default. You can change it below and use your own secret key.
-  # config.secret_key = '7735a7770c224ad54e5d2df45dbb621e855765ae8eaf4af552e182e2776a59643597e09467cab485a10cc56bfebecc88e69dcef66081e5a234162f5f1c7cfe81'
+  config.secret_key = ENV["AV_DEV_KEY_BASE"] if Rails.env.production?
```

Comment:
should be `DEVISE_SECRET_KEY` as per https://github.com/calblueprint/ashby-village/commit/3b873bca82f110114ddc72b776488e61f3a30bab#diff-a8a4112d0760c0fd11f691c6a5fb22adR9